### ISSUE TITLE
find-selection:find-next is slow

### ISSUE THREAD

**Issue by tujoworker** [mark-hahn/find-selection#21]
Hi Mark Hahn,

Thanks for making this package.
I use it on a mac. find-selection:find-previous is super fast, but, find-selection:find-next is boring slow. Any idea why?

**Comment by mark-hahn** [comment]
> Any idea why?

I can't imagine why.  Maybe some other package is interfering?

**Comment by tujoworker** [comment]
Me either. I created a new user on my mac, installed a fresh atom and finally installed only one custom package: find-selection.
The result, same effect.

It takes about 1,5 or 2 sec to find next selection.
To find previous selection happens immediately.

Is there some loop/while function to go through for the package?

**Comment by mark-hahn** [comment]
No loops.  The code is tiny, maybe one or two screens.  Check it out.

How big is the file?  Does this happen on on all files?

**Comment by tujoworker** [comment]
Yes, seems it happens on all files. And no matter what size the files are. Do You are on osx?

**Comment by tujoworker** [comment]
I found out what made select next so slow. I used keybinding "cmd-k" for next, and "shift-cmd-k" for previous. But "cmd-k" is used for a couple of other bindings as well in combination with another keybinding. So "cmd-k" made that atom waits for an additional command. 
Why did i use "cmd-k"? Well, Eclipse is using this by default, so I wish to use same keybindings for things I use often :)
Thanks for the fast response and have a good day!

Solution: 
1. Install https://github.com/aki77/atom-disable-keybindings
2. Simply add "cmd-k" in the package settings "Prefix Keys"
